grpc/src/main/java/com/linecorp/armeria/internal/server/grpc/AbstractServerCall.java

```diff
@@ -424,22 +424,22 @@ private void invokeOnCancel() {
         }
     }
 
-    void onError(Throwable t) {
+    public void onError(Throwable t) {
         if (!closeCalled && !(t instanceof AbortedStreamException)) {
             close(t);
         }
     }
 
-    final void setListener(Listener<I> listener) {
+    public final void setListener(Listener<I> listener) {
         checkState(this.listener == null, "listener already set");
         this.listener = requireNonNull(listener, "listener");
         invokeOnReady();
     }
 
-    abstract void startDeframing();
+    public abstract void startDeframing();
 
     @Nullable
-    final ResponseHeaders responseHeaders() {
+    public final ResponseHeaders responseHeaders() {
```

Comment:
```suggestion
    protected final ResponseHeaders responseHeaders() {
```

Reply:
Done in [fa4d628](https://github.com/line/armeria/commit/fa4d628b0a113437da1d9ebb625332dad3b299be)!